src/wr_logic_longrange/nodes/obstacle_avoidance.py

```diff
@@ -110,9 +119,12 @@ def update_heading(msg: Float64) -> None:
     """
     global cur_heading
     cur_heading = (90 - msg.data) % 360  # Shifting to East
+    # rviz_sim_cur_heading_pub.publish(cur_heading)
+
 
+# calculates difference of angles from -180 to 180 degrees
 def angle_diff(heading1: float, heading2: float) -> float:
-    """
+    """std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
I think the `this_thread::sleep...` was accidentally pasted here.
```suggestion
    """
```